lib/FieldType/Mapper/TextLineFormMapper.php

```diff
@@ -14,23 +14,31 @@
 use EzSystems\RepositoryForms\FieldType\DataTransformer\FieldValueTransformer;
 use EzSystems\RepositoryForms\FieldType\FieldDefinitionFormMapperInterface;
 use EzSystems\RepositoryForms\FieldType\FieldValueFormMapperInterface;
+use Symfony\Component\Form\Extension\Core\Type\IntegerType;
 use Symfony\Component\Form\Extension\Core\Type\TextType;
 use Symfony\Component\Form\FormInterface;
-use Symfony\Component\Form\Extension\Core\Type\IntegerType;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
+/**
+ * FormMapper for ezstring FieldType.
+ */
 class TextLineFormMapper implements FieldDefinitionFormMapperInterface, FieldValueFormMapperInterface
 {
-    /**
-     * @var \eZ\Publish\API\Repository\FieldTypeService
-     */
+    /** @var FieldTypeService */
     private $fieldTypeService;
 
+    /**
+     * @param FieldTypeService $fieldTypeService
+     */
     public function __construct(FieldTypeService $fieldTypeService)
     {
         $this->fieldTypeService = $fieldTypeService;
     }
 
+    /**
+     * @param FormInterface $fieldDefinitionForm
```

Comment:
To be honest, I wouldn't bother with those. They're not adding anything to what the method already says (parameters type and name), and they only repeat the interface.

Some of us choose to use @inheritdoc for that, to show that the phpdoc wasn't forgotten. I prefer not to. Either way, no paraphrasing the prototypes. The best code is the one we don't have to write ;)